ranvier

```diff
@@ -19,9 +19,9 @@ const net = require('net');
 const commander = require('commander');
 const wrap = require('wrap-ansi');
 const argv = require('optimist').argv;
-const longjohn = require('longjohn');
-const pe = require('pretty-error').start();
-pe.skipNodeFiles(); // Ignore native node files in stacktrace.
+//const longjohn = require('longjohn');
+//onst pe = require('pretty-error').start();
```

Comment:
I personally like both pretty error and long stack traces, but they might not be necessary now, especially with Winston, and if we do decide to consider using them, they should probably be extracted to the `Logger` class.